pttrack/models.py

```diff
@@ -71,8 +71,9 @@ def make_filepath(instance, filename):
     while carry_on:
         new_filename = "%s.%s" % (User.objects.make_random_password(48),
                                   filename.split('.')[-1])
-        path = '/'.join([instance.__class__.__name__.lower(),
-                         field_name, new_filename])
+        #path = '/'.join([instance.__class__.__name__.lower(),field_name, new_filename])
+
+        path = new_filename
```

Comment:
Oh yeah, because changing the file paths will break everything and we can't really go back one the system has a particular file path. I feel like breaking it down into folders doesn't really do much since the files names are obscured anyways.